src/app/pages/FeedPage/index.tsx

```diff
@@ -83,7 +83,7 @@ const styles = {
   max-[500px]:w-full
   max-[500px]:h-[56px]
   max-[500px]:justify-center
-  max-[500px]:bg-black
+  max-[500px]:bg-primaryBg-light max-[500px]:dark:bg-primaryBg-dark
```

Comment:
I notice here and in other places for responsive styles you are using very specific values. Why not to use prefixes like build-in breakpoints (here [link](https://tailwindcss.com/docs/responsive-design) to docs). And if they are not sufficient, define own in config